.travis.yml

```diff
@@ -18,63 +18,25 @@ stage: Comprehensive tests
 # Setting sudo to false opts in to Travis-CI container-based builds.
 sudo: false
 
-# The apt packages below are needed for sphinx builds, which can no longer
-# be installed with sudo apt-get.
 # tzdata is included to ensure system leap seconds are up to date.
 addons:
     apt:
         packages:
-            - graphviz
             - language-pack-de
```

Comment:
Looking at this, why is German included here? `language-pack-de` (though arguably not for this PR)

Reply:
I can't remember, I think some tests may force the locale to be ``DE``. I'll try and double check.

Reply:
Yes, some `io.ascii` tests depend on using a difference locale. Because the locale changes how c functions parse strings.

Reply:
Just for the record, the test this is for is test_non_C_locale_with_fast_reader

Reply:
I've added a comment in the Travis file (will push shortly)

Reply:
Done